### PR TITLE
Attribute empty value comes from backend

### PR DESCRIPTION
**Pull request by jmleroux**

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Normalizer/AttributeNormalizer.php [hunk 1]
```diff
@@ -48,4 +49,41 @@ public function supportsNormalization($data, $format = null)
...
+            case 'pim_catalog_text':
+            case 'pim_catalog_textarea':
+                $emptyValue = '';
+                break;
+            case 'pim_catalog_boolean':
+                $emptyValue = false;
+                break;
+            case 'pim_catalog_price_collection':
+                $emptyValue = [];
+                break;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Normalizer/AttributeNormalizer.php]: I prefer a default case that return null

File: src/Pim/Bundle/EnrichBundle/Normalizer/AttributeNormalizer.php [hunk 2]
```diff
@@ -48,4 +49,41 @@ public function supportsNormalization($data, $format = null)
     {
         return $data instanceof AttributeInterface && in_array($format, $this->supportedFormats);
     }
+
+    /**
+     * Get attribute empty value by attribute type
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Normalizer/AttributeNormalizer.php]: I would rather say "Get empty value for specified attribute depending on its type."

File: src/Pim/Bundle/EnrichBundle/Normalizer/AttributeNormalizer.php [hunk 3]
```diff
@@ -48,4 +49,41 @@ public function supportsNormalization($data, $format = null)
...
+     *
+     * @param AttributeInterface $attribute
+     *
+     * @return array|bool|string|null
+     */
+    protected function getEmptyValue($attribute)
+    {
+        $emptyValue = null;
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Normalizer/AttributeNormalizer.php]: Since you're using a `switch` it makes sense to use a `default:` instead of this.

**Comment by phaseinducer** [comment]
GTM
If you wanna avoid the Travis warnings maybe try to rebase your branch on feature branch, all controllers have been modified.

**Pull request merged**